docs/subworkflows/subtype_genome.md

```diff
@@ -12,8 +12,14 @@
 > If a sample cannot be subtyped, it merely passes through the pipeline and is not typed. A log message will instead be displayed notifying the user the sample cannot be typed.
 
 ## Input
-- contigs and associated tags
-- Mash report
+- Contig file (fasta) from the `FinalAssembly` dir
+   - This is the final contig file from the last step in the CleanAssemble workflow (taking into account any skip flags that have been used)
+- Mash report from assembly speciation step in the `Clean Assemble` workflow

```

Comment:
First occurrence is "CleanAssemble", second is "Clean Assemble". Should they be the same?

Reply:
fixed in https://github.com/phac-nml/mikrokondo/pull/66/commits/11de128cfb04339c7178b9f0a6f5e3584ba2c26c